spec/01-abstract.md

```diff
@@ -1,19 +1,22 @@
 
-Today context propagation is implemented in proprietary formats individually by each tracing tool provider. This results in siginficant interoperability problems between tracing tools than manifest in two major ways:
+Today, context propagation is implemented individually by each vendor.
+In multi-vendor environments, this causes interoperability problems, like:
 
-- Traces that are collected by two tracing providers cannot be linked together as there is no unique identifier linking the individual pieces together. 
-- Traces are broken each time they are passed between components monitored by different tracing as there is no uniformly agree set of identifcation taht is forwarded.
+- Traces that are collected by different tracing vendors cannot be linked together as there is no shared unique identifier.
+- Traces that cross boundaries between different tracing vendors break as there is no uniformly agreed set of identification that is forwarded.
+- Cloud platform vendors, as well as hardware vendors, cannot guarantee to support context propagation as there is no standardized way of context propagation.
 
-In the past this problem did not have significant impact as most applications were under monitored by a single tracing tool. Today, an increasing number of applications are highly distributed and leverage an increasing number of cloud platform and middleware services. Most of these services come with their own tracing capababilities which results in big interoperabiity problems and lack of visibility for developers and appliation operators.
+In the past, these problems did not have a significant impact as most applications were monitored by a single tracing tool and stayed within the boundaries of a single platform provider. Today, an increasing number of applications are highly distributed and leverage multiple middleware services and cloud platforms.
 
 ## Solution
 
-The trace context specification defines a universally agreed format for the exchange of context propagation data - referred to as trace context. Trace context solved the problems described above by
+The trace context specification defines a universally agreed-upon format for the exchange of context propagation data - referred to as *trace context*. Trace context solves the problems described above by
 
-- providing a unique identifier for individual tracing allowing traces of multiple providers being linked together and 
-- providing an agreed-upon mechanism to forward vendor-specific trace data avoid broken tracing when multiple trace tools particpate in a single transaction. 
+- providing a unique identifier for individual tracing, allowing traces of multiple providers to be linked together and 

```

Comment:
> individual tracing

should be "traces" I think, though this was incorrect in the original text as well